tests/mail/tests.py

```diff
@@ -1106,50 +1108,58 @@ def test_backend_arg(self):
 
     @override_settings(
         EMAIL_BACKEND="django.core.mail.backends.locmem.EmailBackend",
-        ADMINS=[("nobody", "nobody@example.com")],
-        MANAGERS=[("nobody", "nobody@example.com")],
+        ADMINS=["nobody@example.com"],
+        MANAGERS=["nobody@example.com"],
     )
     def test_connection_arg(self):
         """Test connection argument to send_mail(), et. al."""
         mail.outbox = []
 
         # Send using non-default connection
-        connection = mail.get_connection("mail.custombackend.EmailBackend")
```

Comment:
Reviewer note: if you're seeing a huge change here, suggest enabling "hide whitespace" in GitHub's review options.